de.properties.tmpl

```diff
@@ -0,0 +1,95 @@
+org.iplantc.discoveryenvironment.workspace.rootAppCategory = {{ key (printf "%s/de/root-app-category" $base) }}
+org.iplantc.discoveryenvironment.workspace.defaultAppCategories = {{ key (printf "%s/de/default-app-categories" $base) }}
+org.iplantc.discoveryenvironment.workspace.defaultBetaAppCategoryId = {{ key (printf "%s/de/default-beta-app-category" $base) }}
+org.iplantc.discoveryenvironment.workspace.defaultHpcAppCategoryId = {{ key (printf "%s/de/default-hpc-app-category" $base) }}
+org.iplantc.discoveryenvironment.workspace.defaultTrashAppCategoryId = {{ key (printf "%s/de/default-trash-app-category" $base) }}
+org.iplantc.discoveryenvironment.workspace.ontologyAttrs = {{ key (printf "%s/de/ontology-attributes" $base) }}
+org.iplantc.discoveryenvironment.workspace.metadata.beta.attr.iri = {{ key (printf "%s/de/beta-attr-iri" $base) }}
+org.iplantc.discoveryenvironment.workspace.metadata.beta.attr.label = {{ key (printf "%s/de/beta-attr-label" $base) }}
+org.iplantc.discoveryenvironment.workspace.metadata.beta.value = {{ key (printf "%s/de/beta-value" $base) }}
+org.iplantc.discoveryenvironment.workspace.metadata.beta.unit = {{ key (printf "%s/de/beta-unit" $base) }}
+org.iplantc.discoveryenvironment.workspace.apps.cardUrl = {{ key (printf "%s/de/card-url" $base) }}
+org.iplantc.discoveryenvironment.workspace.apps.cardUrlOptions = {{ key (printf "%s/de/card-url-options" $base) }}
+org.iplantc.discoveryenvironment.contextMenu.enabled = {{ key (printf "%s/de/context-menu-enabled" $base) }}
+org.iplantc.communitydata.path = {{ key (printf "%s/de/community-data-path" $base) }}
+org.iplantc.irodshome.path = {{ key (printf "%s/de/irods-home" $base) }}
+org.iplantc.basetrash.path = {{ key (printf "%s/de/trash-path" $base) }}
+org.iplantc.discoveryenvironment.maintenance-file = {{ key (printf "%s/de/maintenance-file" $base) }}
+org.iplantc.discoveryenvironment.local-ip-ranges = {{ key (printf "%s/de/local-ip-ranges" $base) }}
+org.iplantc.discoveryenvironment.environment.name = {{ key (printf "%s/de/environment-name" $base) }}
+org.iplantc.discoveryenvironment.environment.prod-deployment = {{ key (printf "%s/de/prod-deployment" $base) }}
+org.iplantc.discoveryenvironment.notifications.poll-interval = {{ key (printf "%s/de/poll-interval" $base) }}
+org.iplantc.discoveryenvironment.notification.amqp.uri = {{ key (printf "%s/de/amqp-de-uri" $base) }}
+org.iplantc.discoveryenvironment.notification.amqp.exchange.name = {{ key (printf "%s/de/amqp-de-name" $base) }}
+org.iplantc.discoveryenvironment.notification.amqp.exchange.durable = {{ key (printf "%s/de/amqp-de-durable" $base) }}
+org.iplantc.discoveryenvironment.notification.amqp.exchange.auto-delete = {{ key (printf "%s/de/amqp-de-auto-delete" $base) }}
+org.iplantc.discoveryenvironment.analysis.support.user = {{ key (printf "%s/de/support-user" $base) }}
+org.iplantc.discoveryenvironment.support = {{ key (printf "%s/terrain/base" $base) }}/support-email
```

Comment:
Similar question as my review for cyverse-de/ansible#25: Will we need to update these new settings and the old `de.properties.j2` ansible template when we add or remove UI configs now?